src/librustdoc/html/static/source-script.js

```diff
@@ -231,7 +232,7 @@ var handleSourceHighlight = (function() {
     };
 }());
 
-window.addEventListener("hashchange", function() {
+window.addEventListener("hashchange", function(ev) {
```

Comment:
This is actually incorrect. `highlightSourceLines` interprets `!ev` to mean "the wasn't triggered by a mouse click, so scroll the relevant lines into view."

Since highlightSourceLines doesn't use any of the properties of `ev`, we should turn this into a boolean `scroll` param to make its purpose more explicit, and always supply the param.

Reply:
Done!